includes/data/connection/class-product-connection-resolver.php

```diff
@@ -34,6 +33,13 @@ class Product_Connection_Resolver extends AbstractConnectionResolver {
 	 */
 	protected $post_type;
 
+	/**
+	 * The instance of the class that helps filtering with the product attributes lookup table.
+	 *
+	 * @var Filterer
```

Comment:
```suggestion
	 * @var \Automattic\WooCommerce\Internal\ProductAttributesLookup\Filterer
```

Add `|\stdClass` ,if this prop is overloadable.

Reply:
It's not overloadable as fair as I know.